### ISSUE TITLE
How to screencast a url?

### ISSUE THREAD

**Issue by ariel-frischer** [blueimp/wdio#25]
Any examples with screenshots or screencasting? Also does screencasting support recording with an alpha channel? (like to webm or mov format)?

**Comment by blueimp** [comment]
Yes you can create a screen recording of loading a specific URL.
While the project's purpose is to provide a Docker setup for [Webdriver.io](https://webdriver.io/), it also comes with integration for video recording.

You can enable video recordings in the browser configs. For Firefox it's enabled by default:
https://github.com/blueimp/wdio/blob/77b7c9518bc377334ac497e83ce0e7265c682ca9/conf/firefox.js#L18

There's in [example in the test specs](https://github.com/blueimp/wdio/blob/77b7c9518bc377334ac497e83ce0e7265c682ca9/test/specs/01-session.js#L35) how to save and diff screenshots.

The package providing the screen recording and diffing integration has more usage examples:
https://github.com/blueimp/wdio-screen-commands#usage

There is no support for alpha channel screen recordings.
